xapian-letor/ranker/listmle_ranker.cc

```diff
@@ -133,10 +134,14 @@ ListMLERanker::train(const vector<vector<FeatureVector>>& training_data)
 	throw InvalidArgumentError("Cannot train: no training data");
     int feature_cnt = training_data[0][0].get_fcount();
 
-    // Initialize the parameters for neural network
+    // initialize the parameters for neural network with Xavier initialization.
+    // construct a trivial random generator engine:
```

Comment:
If a comment is a sentence, it really should start with a capital letter (like it did before this change).

Reply:
fixed in a fixup commit.